content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -521,33 +521,33 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
 
-## Citadel is not behaving properly {#repairing-citadel}
+## Istiod certificate management is not behaving properly {#repairing-istiod}
```

Comment:
#repairing-istiod is too broad. Does the text have to be consistent with label? If not, I think #repairing-citadel is ok. 

We still need a name to refer to Istio CA in Istiod process. I think it is ok to keep the name "Citadel", even though Citadel is part of Istiod process. @myidpt What do you think?